projects/plugins/beta/class-jetpack-beta-admin.php

```diff
@@ -168,12 +168,30 @@ public static function to_test_content() {
 			case 'pr':
 				return self::to_test_pr_content( $branch );
 			case 'master':
+				return self::to_test_general_rules_content();
 			case 'rc':
 				return self::to_test_file_content();
 		}
 		return null;
 	}
 
+	/**
+	 * General rules and recommendations for new Beta Testers.
+	 * Displayed when no specific branch is picked.
+	 *
+	 * @since 2.5.0
+	 */
+	public static function to_test_general_rules_content() {
+		$test_rules = WP_PLUGIN_DIR . '/' . Jetpack_Beta::get_plugin_slug() . '/docs/testing/testing-tips.md';
```

Comment:
I think we'll need to move that file inside the Jetpack Beta plugin, so it can be shipped with it.

Reply:
13c475b